tests/common/devices.py

```diff
@@ -232,14 +232,20 @@ def get_crm_resources(self):
 
         return result
 
-    def get_pmon_daemon_list(self):
+    def get_pmon_daemon_states(self):
         """
-        @summary: get pmon daemon list from the config file (/usr/share/sonic/device/{platform}/{hwsku}/pmon_daemon_control.json)
+        @summary: get state list of daemons from pmon docker.
+                  Referencing (/usr/share/sonic/device/{platform}/{hwsku}/pmon_daemon_control.json)
```

Comment:
Per below discussion:
```suggestion
                  Referencing (/usr/share/sonic/device/{platform}/pmon_daemon_control.json)
```

Reply:
Thanks! Fixed now!